otx/algorithms/detection/adapters/mmdet/pipelines.py

```diff
@@ -0,0 +1,139 @@
+"""Collection Pipeline for detection task."""
```

Comment:
Similar comments with above. I'd like to suggest put this in `data` folder together with dataset files.

Reply:
I moved it inside `mmdet/utils`. Thanks for the nice comment.